SwedbankPaySDK/Classes/SwedbankPayWebViewController.swift

```diff
@@ -169,21 +169,21 @@ extension SwedbankPayWebViewController : WKNavigationDelegate {
         }
     }
     
-    private func decidePolicyFor(url: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
+    private func decidePolicyFor(navigationAction: WKNavigationAction, url: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 145 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)